curriculum/challenges/english/25-front-end-development/lab-event-hub/66ebd4ae2812430bb883c787.md

```diff
@@ -177,7 +177,7 @@ Both of the `article` elements inside the `#upcoming-events` section should have
 
 ```js
 const pElements = document.querySelectorAll('#upcoming-events article');
-assert.isNotEmpty(articles);
+assert.isNotEmpty(pElements);
```

Comment:
this is not the change I asked you to do, please use the more descriptive variable name `articles`. `pElements` does not make sense, you are not selecting `p` elements